bin/get-url.js

```diff
@@ -0,0 +1,43 @@
+/* eslint no-console: ["error", { allow: ["error"] }] */
+const debug = require('debug')('get-url')
+const { prompt } = require('enquirer')
+const isUrl = require('is-url-superb')
+const { Utils } = require('../index')
+
+const validate = function(url) {
```

Comment:
how about we change this to function declaration?
e.g.

```
function validate(url) 
...
```

Reply:
Good!